_jobs/software-engineer.md

```diff
@@ -21,41 +23,47 @@ We are looking for a Software Engineer to support a range of new and ongoing cli
 - Document your work, including asynchronous communications, to inform internal and client teams
 - Coordinate with product, design, and engineering to deliver delightful products and services
 - Pair program, code review, contribute to architecture decisions, help create standard patterns and practices for development and DevOps
-- Self-direct and time-manage across multiple client projects with varying levels of complexity, demand, and constraint
+- Self-direct and time-manage across multiple client and internal projects with varying levels of complexity, demand, and constraint
 
-## Preferred qualifications
+## Preferred Qualifications
 
 - Experience working with or in proximity to government
 - Experience using modern, high-level programming languages such as Python and JavaScript to build web applications, API services, or similar
 - Experience using Docker and containers to manage reproducible environments
-- Experience deploying production applications and services to cloud environments such as AWS and GCP
-- Experience collaborating with product managers and remote teams
+- Experience deploying and managing production applications and services in cloud environments such as Azure and GCP, preferably with Infrastructure as Code systems such as Terraform
+- Experience collaborating with product managers, UI/UX designers, and remote teams
 - Knowledge of Git / GitHub based development workflows and open source project management
 - A love of public transportation
-- A personal and professional commitment to bring anti-racist principles into development practices
+- A personal and professional commitment to bring anti-racist principles into your work
+- Preference given to candidates located in California, and specifically Los Angeles
```

Comment:
LA..city? County? Does it matter? @thekaveman

Reply:
I think we'll leave this one more open. It doesn't matter as much, more so that they are local-ish.